src/glib/glib.c

```diff
@@ -11,6 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <stddef.h>
```

Comment:
It got lost during PR #778. The change here looks good to me. 

Since we need this header (so far) only because of "offsetof" that is only in this C file, it could simply be included here, but I will let @levb decide if he would rather have it in "glibp.h".